lms/djangoapps/mobile_api/models.py

```diff
@@ -1,3 +1,23 @@
 """
 A models.py is required to make this an app (until we move to Django 1.7)
 """
+# pylint: disable=no-member
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class MobileApiConfig(ConfigurationModel):
+    """Configuration for the video upload feature."""
+    video_profiles = TextField(
+        blank=True,
+        help_text="A comma-separated list of names of profiles when requesting video data from VAL."
```

Comment:
This should be described in terms of the interface, not the implementation.

"A comma-separated list of names of profiles to include for videos returned from the mobile API."